common/hooks/pre-configure/02-script-wrapper.sh

```diff
@@ -8,8 +8,11 @@ generic_wrapper() {
 	[ ! -x ${XBPS_CROSS_BASE}/usr/bin/${wrapper} ] && return 0
 	[ -x ${XBPS_WRAPPERDIR}/${wrapper} ] && return 0
 
-	echo "#!/bin/sh" >> ${XBPS_WRAPPERDIR}/${wrapper}
-	echo "exec ${XBPS_CROSS_BASE}/usr/bin/${wrapper} --prefix=${XBPS_CROSS_BASE}/usr \"\$@\"" >> ${XBPS_WRAPPERDIR}/${wrapper}
+	cat >>${XBPS_WRAPPERDIR}/${wrapper}<<_EOF
```

Comment:
I think it's cleaner to append minus `-` before `_EOF`,
we can indent the here doc with tab then.

Reply:
No, that is inconsistent with all other usages in the file.